pallets/liquidation/src/lib.rs

```diff
@@ -298,6 +300,7 @@ impl<T: Config> Pallet<T> {
         Ok(result)
     }
 
+    #[allow(clippy::type_complexity)]
```

Comment:
we can allow `type_complexity` for all, this should have been done in `make lint`